server/proxy/install.go

```diff
@@ -17,23 +17,24 @@ import (
 	"github.com/mattermost/mattermost-plugin-apps/utils"
 )
 
-func (p *Proxy) InstallApp(client mmclient.Client, sessionID string, cc *apps.Context, trusted bool, secret string) (*apps.App, string, error) {
+// InstallApp installs an App.
+//  - cc is the Context that will be passed down to the App's OnInstall callback.
+func (p *Proxy) InstallApp(in Incoming, cc apps.Context, appID apps.AppID, trusted bool, secret string) (*apps.App, string, error) {
 	conf, _, log := p.conf.Basic()
-	log = log.With("app_id", cc.AppID)
-	m, err := p.store.Manifest.Get(cc.AppID)
+	log = log.With("app_id", appID)
+	m, err := p.store.Manifest.Get(appID)
 	if err != nil {
 		return nil, "", errors.Wrap(err, "failed to find manifest to install app")
 	}
-
```

Comment:
I don't have an objections on changing the spacing in a PR, but it was consistent before this one. What made you change it in occasional places?

Reply:
Accidents mostly, sometimes reflexive attempts to fix the typography. Not intentional, please point out and I'll remove, I often review with -w to skip the struct alignment changes.